docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -501,52 +623,59 @@ node_labels:
 
 At a first glance, Trusted Clusters in combination with RBAC may seem
 complicated. However, it is based on certificate-based SSH authentication
-which is fairly easy to reason about:
+which is fairly easy to reason about.
 
 One can think of an SSH certificate as a "permit" issued and time-stamped by a
 certificate authority. A certificate contains four important pieces of data:
 
-- List of allowed UNIX logins a user can use. They are called "principals" in the certificate.
-- Signature of the certificate authority who issued it (the *auth* server)
-- Metadata (certificate extensions): additional data protected by the signature above. Teleport uses the metadata to store the list of user roles and SSH
+- List of allowed Unix logins a user can use. They are called "principals" in
+  the certificate.
+- Signature of the certificate authority that issued it (the Teleport Auth Service)
+- Metadata (certificate extensions): additional data protected by the signature
+  above. Teleport uses the metadata to store the list of user roles and SSH
   options like "permit-agent-forwarding".
 - The expiration date.
 
 Try executing `tsh status` right after `tsh login` to see all these fields in the
 client certificate.
 
-When a user from "root" tries to connect to a node inside "leaf", her
-certificate is presented to the auth server of "leaf" and it performs the
+When a user from the root cluster tries to connect to a node inside "leaf", the user's
+certificate is presented to the Auth Service of "leaf" and it performs the
 following checks:
 
-- Checks that the certificate signature matches one of the trusted clusters.
+- Checks that the certificate signature matches one of the Trusted Clusters.
 - Tries to find a local role that maps to the list of principals found in the certificate.
-- Checks if the local role allows the requested identity (UNIX login) to have access.
+- Checks if the local role allows the requested identity (Unix login) to have access.
```

Comment:
Nit: in the PRs I've reviewed today, we've used all of these interchangeably:

- Unix login
- OS login
- Host login

Should we standardize on a term here?

Reply:
I'm not sure we need to have a standard for this unless this is a Teleport-specific technical term. We use `logins` in user/role resource definitions, but the docs also sometimes use the term outside of that context. I'll add it to my list of possible topics to include in a style guide.